Makefile

```diff
@@ -30,11 +30,14 @@ clean: ## remove generated byte code, coverage reports, and build artifacts
 
 extract_translations: ## extract strings to be translated, outputting .po files
 	cd $(WORKING_DIR) && i18n_tool extract
-	mv $(EXTRACTED_DJANGO) $(EXTRACTED_TEXT)
-	tail -n +20 $(EXTRACTED_DJANGOJS) >> $(EXTRACTED_TEXT)
-	rm $(EXTRACTED_DJANGOJS)
-	sed -i'' -e 's/nplurals=INTEGER/nplurals=2/' $(EXTRACTED_TEXT)
-	sed -i'' -e 's/plural=EXPRESSION/plural=\(n != 1\)/' $(EXTRACTED_TEXT)
+	mv $(EXTRACTED_DJANGO_PARTIAL) $(EXTRACTED_DJANGO)
+	# Safely concatenate djangojs if it exists
+	( test -f $(EXTRACTED_DJANGOJS_PARTIAL) && \
+	  msgcat $(EXTRACTED_DJANGO) $(EXTRACTED_DJANGOJS_PARTIAL) -o $(EXTRACTED_DJANGO) \
+	) || ! test -f $(EXTRACTED_DJANGOJS_PARTIAL)
```

Comment:
perform the concatenation if-and-only-if the file exists. The syntax might look redundant, but its actually safer than the below syntax that absorbs errors generated by `msgcat`

```shell
test -f $(EXTRACTED_DJANGOJS_PARTIAL) && msgcat $(EXTRACTED_DJANGO) $(EXTRACTED_DJANGOJS_PARTIAL) -o $(EXTRACTED_DJANGO) || true
```

Reply:
@shadinaif, why is `|| ! test -f $(EXTRACTED_DJANGOJS_PARTIAL)` safer than `|| true`? It's doing the same thing, but using `|| true` makes it obvious that we want to ignore potential non-zero exit codes. To me, using the reversed file check gives a feeling that something is missing after this statement.

Reply:
It's not the same @Agrendalath , because with `|| true` the statement will absorb any error raised by `msgcat`. But since you found it complicated then I must admit that it is indeed. I've changed it to an `if` statement that is more readable. Can you please check it again? thanks a lot for your time and efforts 🌷